### ISSUE TITLE
Broken Chunks?

### ISSUE THREAD

**Issue by Jujuchiii** [CorgiTaco/BYG#78]
In this scenarios, I'm simply using Nature's Compass Mod, alongside BYG. For example, I'll search for a biome in Nature's Compass and then teleport the coordinates it displays, but since they're unloaded they kind of make the chunk below the surface, which is really odd.

Not sure if this is related to BYG no generating well? Because I've never had this problem with other Biome Mods.

Here are some examples of the issue I encounter when teleporting to unloaded chunks/biomes...

![2020-08-29_23 41 54](https://user-images.githubusercontent.com/53573738/91650793-09bfbd00-ea52-11ea-9b07-b3e4746aa360.png)
![2020-08-29_23 42 01](https://user-images.githubusercontent.com/53573738/91650794-0cbaad80-ea52-11ea-800a-c822224b1ff0.png)

**Comment by CorgiTaco** [comment]
What mc version, api, and byg version? What world type?

**Comment by Jujuchiii** [comment]
I'm currently using...

MC Ver. 1.16.1
BYG Ver. 1.0.16 for MC 1.16.1
I think the normal world type. Im not sure what the base vanilla one is
called. I simply used the world I loaded on singleplayer and imported it
into my server.

I'm thinking a cause of it is that I recently downgraded from 1.16.2 since
me and others were getting consistent crashes here and there when we
played. 1.16.1 was much more stable but I kept the 1.16.2 world, which is
the one you see in the screenshots.

But I have no clue haha.
